### DISCUSSION TITLE
Diagramm Anzeige auf 2,42 Zoll Display

### DISCUSSION THREAD

**Discussion by jblond4711** [tbnobody/OpenDTU#1604] [General]
Hi,

mal eine Frage an die Benutzer mit einem 2,42 Zoll Display. Wird bei euch das Diagramm komplett angezeigt?
Bei mir ist das -> etwas verschoben so das das > an der linken Seite angezeigt wird.
Ist das Bildschirmschoner aktiv wird das > natürlich noch weiter verschoben.
Das gleiche habe ich auch mit der kWh Anzeige vom Gesamtwert da rutscht auch immer ein Teil auf die Linke Seite aber erst seitdem es mehr als 1000 kWh sind.

**Comment by tbnobody** [comment]
hab mein display heute bekommen und hatte das gleiche thema. das liegt am verwendeten treiber. werd demnächst ne firmware für den ssd1309 chip releasen. dann gibts das problem nicht mehr.

**Comment by jblond4711** [comment]
Das hört sich gut an, vielen Dank.

**Comment by leoman4** [comment]
Same bei mir. Gibt es bald einen neuen Firmware?

**Comment by jblond4711** [comment]
Hi,
das Problem ist doch schon lange behoben. Stell mal bitte bei dir den Displaytyp auf 4.

**Reply by leoman4** [reply]
👌🏽 Danke. Ihr macht wirklich gute Arbeit
